sharepoint/README.md

```diff
@@ -0,0 +1,52 @@
+# SharePoint Data Connector Quickstart
+
+Use the SharePoint Data Connector to query documents stored in SharePoint from Spice.
+
+## Requirements
+
+- [Spice CLI](https://docs.spiceai.org/getting-started) installed.
+- Access to a SharePoint instance, and know the associated tenant ID.
+  - (Optional) Upload relevant documents to the user's root sharepoint directory
+
+## Follow these steps
+
+**Step 1.** Login in Sharepoint via Spice CLI.
+
+```bash
+export TENANT_ID=<YOUR_TENANT_ID>
+spice login sharepoint --tenant-id $TENANT_ID --client-id f2b3116e-b4c4-464f-80ec-73cd9d9886b4
```

Comment:
Will this client ID work for all tenants?

Reply:
Yes, this is the Spice hosted method: https://docs.spiceai.org/components/data-connectors/sharepoint#default-spice-application